content/working_with/console/widgets/deploymentInfo.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: bt_wiki
+title: Deployment Info
+category: Cloudify Console
+draft: false
+---
+Displays the following information about a specific deployment: 
+
+* **Name**
+* **Description**
+* **Visibility level**
+* **Blueprint name**
+* **Site name**
+* **Creation time**
+* **Last update time**
+* **Creator username**
+
+
+![deployment-info]( /images/ui/widgets/deployment-info.png )
+
+#### Widget Settings
+* `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
+* `Show blueprint name` - Defines whether deployment's blueprint name should be displayed. Default: On
+* `Show site name` - Defines whether deployment's site name should be displayed. Default: On
+* `Show created date` - Defines whether deployment's created date should be displayed
+* `Show updated date` - Defines whether deployment's last update date should be displayed. Default: On
+* `Show creator` - Defines whether deployment's creator username should be displayed. Default: On
+* `Show node instances status` - Defines whether deployment's node instances status should be displayed. Default: On
```

Comment:
I don't know which style is correct (maybe both?) but there is inconsistency in apostrophes usage - see line 22 (widget`s) versus other lines (deployment's)

Reply:
Fixed.